config/develop/tower-dev-certificate.yaml

```diff
@@ -0,0 +1,11 @@
+template_path: remote/acm-certificate.yaml
+stack_name: tower-dev-certificate
+parameters:
+  Department: IBC
+  Project: Infrastructure
+  OwnerEmail: nextflow-admins@sagebase.org
+  DnsDomainName: sagebionetworks.org
```

Comment:
hmm, isn't this domain already registered?  I think you need to setup your own unique domain then redirect to it from sagebionetworks.org

Reply:
@zaro0508 Do you see the `DnsSubDomainName` directly underneath it? These two are being joined together. This is what the template contents looks like that does that:
```
  Certificate:
    Type: AWS::CertificateManager::Certificate
    Properties:
      DomainName: !Join
        - '.'
        - - !Ref DnsSubDomainName
          - !Ref DnsDomainName
      DomainValidationOptions:
        - DomainName: !Join
            - '.'
            - - !Ref DnsSubDomainName
              - !Ref DnsDomainName
          ValidationDomain: !Ref DnsDomainName
      ValidationMethod: "DNS"
```

Reply:
@xschildw Can you verify that I am setting this up correctly?

Reply:
Looks good, wondering about that ValidationDomain option in the template itself...

Reply:
The [doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-certificatemanager-certificate-domainvalidationoption.html) says that the `ValidationDomain` is:

> The domain name to which you want ACM to send validation emails. This domain name is the suffix of the email addresses that you want ACM to use. This must be the same as the DomainName value or a superdomain of the DomainName value. For example, if you request a certificate for testing.example.com, you can specify example.com as this value. In that case, ACM sends domain validation emails to the following five addresses:
> 
> admin@example.com
> 
> administrator@example.com
> 
> hostmaster@example.com
> 
> postmaster@example.com
> 
> webmaster@example.com

Sounds pretty benign